src/groups/connections.jl

```diff
@@ -0,0 +1,174 @@
+const CartanSchoutenZeroGroup{𝔽,M} = ConnectionManifold{𝔽,M,CartanSchoutenZero}
+
+"""
+    exp!(M::ConnectionManifold{𝔽,<:AbstractGroupManifold{𝔽},<:AbstractCartanSchoutenConnection}, q, p, X) where {𝔽}
+
+Compute the exponential map on the [`ConnectionManifold`](@ref) `M` with a Cartan-Schouten
+connection. See Sections 5.3.2 and 5.3.3 of [^Pennec2020] for details.
+
+[^Pennec2020]:
+    > X. Pennec and M. Lorenzi, “5 - Beyond Riemannian geometry: The affine connection
+    > setting for transformation groups,” in Riemannian Geometric Statistics in Medical Image
+    > Analysis, X. Pennec, S. Sommer, and T. Fletcher, Eds. Academic Press, 2020, pp. 169–229.
+    > doi: 10.1016/B978-0-12-814725-2.00012-1.
+"""
+function exp!(
+    M::ConnectionManifold{𝔽,<:AbstractGroupManifold{𝔽},<:AbstractCartanSchoutenConnection},
+    q,
+    p,
+    X,
+) where {𝔽}
+    Y = inverse_translate_diff(M.manifold, q, p, X)
+    return compose!(M.manifold, q, p, group_exp(M.manifold, Y))
+end
+
+"""
+    log!(M::ConnectionManifold{𝔽,<:AbstractGroupManifold{𝔽},<:AbstractCartanSchoutenConnection}, Y, p, q) where {𝔽}
+
+Compute the logarithmic map on the [`ConnectionManifold`](@ref) `M` with a Cartan-Schouten
+connection. See Sections 5.3.2 and 5.3.3 of [^Pennec2020] for details.
+
+[^Pennec2020]:
+    > X. Pennec and M. Lorenzi, “5 - Beyond Riemannian geometry: The affine connection
+    > setting for transformation groups,” in Riemannian Geometric Statistics in Medical Image
+    > Analysis, X. Pennec, S. Sommer, and T. Fletcher, Eds. Academic Press, 2020, pp. 169–229.
+    > doi: 10.1016/B978-0-12-814725-2.00012-1.
+"""
+function log!(
+    M::ConnectionManifold{𝔽,<:AbstractGroupManifold{𝔽},<:AbstractCartanSchoutenConnection},
+    Y,
+    p,
+    q,
+) where {𝔽}
+    pinvq = compose(M.manifold, inv(M.manifold, p), q)
+    group_log!(M.manifold, Y, pinvq)
+    return translate_diff!(M.manifold, Y, p, Identity(M.manifold, p), Y)
+end
+
+"""
+    vector_transport_to(M::CartanSchoutenMinusGroup, ::Identity, X, q, ::ParallelTransport)
+
+Transport tangent vector `X` identity on the group manifold with the
+[`CartanSchoutenMinus`](@ref) connection to point `q`. See [^Pennec2020] for details.
+
+[^Pennec2020]:
+    > X. Pennec and M. Lorenzi, “5 - Beyond Riemannian geometry: The affine connection
+    > setting for transformation groups,” in Riemannian Geometric Statistics in Medical Image
+    > Analysis, X. Pennec, S. Sommer, and T. Fletcher, Eds. Academic Press, 2020, pp. 169–229.
+    > doi: 10.1016/B978-0-12-814725-2.00012-1.
+"""
+vector_transport_to(M::CartanSchoutenMinusGroup, Y, ::Identity, X, q, ::ParallelTransport)
+
+function vector_transport_to!(M::CartanSchoutenMinusGroup, Y, p, X, q, ::ParallelTransport)
+    return inverse_translate_diff!(M.manifold, Y, q, p, X, LeftAction())
+end
+
+"""
+    vector_transport_to(M::CartanSchoutenPlusGroup, ::Identity, X, q, ::ParallelTransport)
```

Comment:
Well this is – both in the documentation and the signature for the documentation, still done for `::Identity`, while the generic implementation is still done for a.. `p`, so my question remains whether this might be inconsistent?

Reply:
Right, it should be fixed now.